make_queue/templates/make_queue/machine/machine_edit.html

```diff
@@ -59,6 +61,13 @@ <h1 class="ui centered header">
                 </div>
             </div>
 
+            <div class="ui two fields">
+                <div class="ui text field">
+                    <label>{{ form.stream_name.label }}</label>
+                    {{ form.stream_name }}
+                </div>
+            </div>
```

Comment:
I think that ideally it should not be necessary to specify each field in this way. I believe this can be solved with a for loop and a separate template for form fields. I don't think I should correct it in this PR though.

Reply:
You are welcome to solve it :)